__tests__/database.test.js

```diff
@@ -166,6 +166,62 @@ describe("GET comments", () => {
     })
 })
 
+
+describe("POST comments", () => {
+    const matchCommentObject = {
+        comment_id: expect.any(Number),
+        votes: 0,
+        created_at: expect.any(String),
+        author: 'rogersop',
+        body: 'Amazing article',
+        article_id: 2
+      }
+      const validComment = {
+        username: 'rogersop',
+        body: 'Amazing article'
+    }
+    test("POST /api/articles/2/comments with existing user and valid body returns a comment", () => {
+        return request(app)
+        .post('/api/articles/2/comments')
+        .send(validComment)
+        .expect(201)
+        .then(({body}) => {
+            expect(body[0]).toMatchObject(matchCommentObject)
```

Comment:
We shouldn't be getting back an array here, as it's a single thing we get back it should come back as an object.